specification/deviceupdate/resource-manager/Microsoft.DeviceUpdate/preview/2022-12-01-preview/deviceupdate.json

```diff
@@ -0,0 +1,1997 @@
+          "Checks whether instance exists.": {
+            "$ref": "./examples/Instances/Instances_Head.json"
+          }
+        }
+      },
+      "put": {
+        "description": "Creates or updates instance.",
+        "operationId": "Instances_Create",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/AccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/InstanceNameParameter"
+          },
+          {
+            "name": "Instance",
+            "description": "Instance details.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Instance"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Async operation to create or update Instance was created.",
+            "schema": {
+              "$ref": "#/definitions/Instance"
+            }
+          },
+          "default": {
+            "description": "Error response describing the reason for operation failure.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Creates or updates Instance": {
+            "$ref": "./examples/Instances/Instances_Create.json"
+          }
+        }
+      },
+      "delete": {
+        "description": "Deletes instance.",
+        "operationId": "Instances_Delete",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/AccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/InstanceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Instance was deleted."
+          },
+          "202": {
+            "description": "Async operation to delete Instance was created.",
+            "headers": {
+              "Azure-AsyncOperation": {
+                "type": "string",
+                "description": "Operation Status Location URI"
+              }
+            }
```

Comment:
Above, "x-ms-running-operation-options" is configured to be using "Location" header for async response. However, you are adding "Azure-AsyncOperation" header here in 202 response. It should be same at both the places, Location or Azure-AsyncOperation.

Reply:
Thanks, fixed!

Reply:
@visingla-ms @xiaoxuqi-ms I don't think I have permissions to remove the label. Could you please remove the label and approve? Thanks